frontend/src/bundles/common/components/button/button.tsx

```diff
@@ -1,12 +1,33 @@
-import { Button as LibraryButton } from '@chakra-ui/react';
+import {
+    type SystemStyleObject,
+    Button as LibraryButton,
+} from '@chakra-ui/react';
 
 type Properties = {
     label: string;
     type?: 'button' | 'submit';
+    variant?: string;
+    size?: 'md' | 'lg';
+    isDisabled?: boolean;
+    sx?: SystemStyleObject;
 };
 
-const Button: React.FC<Properties> = ({ type = 'button', label }) => (
-    <LibraryButton type={type} width="full">
+const Button: React.FC<Properties> = ({
+    type = 'button',
+    label,
+    variant = 'solid',
+    size = 'md',
+    isDisabled = false,
+    sx = {},
+}) => (
```

Comment:
I think that default parameters should be placed at the end. For example :
({
 	label,		 
	type = 'button',
	variant = 'solid',
	size = 'md',
	isDisabled = false,
	sx = {},
})

Reply:
required props should go first

Reply:
Done